CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 
 - [BUGFIX] Operator: Fix relabel_config directive for PodLogs resource (@hjet)
 
+- [BUGFIX] Update go-kit logger package to remove debug logs (@mapno)
```

Comment:
This was introduced prior to a release so a CHANGELOG entry isn't necessary, can you remove?

Reply:
Oh that's right. Removed